diff --git a/package/squeezelite/squeezelite.hash b/package/squeezelite/squeezelite.hash
index 0b21d861b9..2eb8f6fda0 100644
--- a/package/squeezelite/squeezelite.hash
+++ b/package/squeezelite/squeezelite.hash
@@ -1,3 +1,3 @@
 # Locally calculated
-sha256 7f0342f4943d3c3498d930060f5a1e8d63ccb7282b9c21f964926e310449d592 squeezelite-71c012ad9ba102feb95823b7b9dc17e5305689c7.tar.gz
-sha256 dc01c3da33fea11ed918f9e437b88baa59c70401aab0dad747e33dd9e8035706 LICENSE.txt
+sha256 abf57c73b83ade5e6266143ea11c049101f172f7f321104d0270fb0bd7340d32 squeezelite-3fbf7ea5288fad99e76800ee566771bbbce49299.tar.gz
+sha256 2786080f6f390bd8d925738414edb546cb0d587225fd4ef7909e1ab6b887454e squeezelite-3fbf7ea5288fad99e76800ee566771bbbce49299/LICENSE.txt
diff --git a/package/squeezelite/squeezelite.mk b/package/squeezelite/squeezelite.mk
index b0347c4df9..82d5ec6c65 100644
--- a/package/squeezelite/squeezelite.mk
+++ b/package/squeezelite/squeezelite.mk
@@ -4,7 +4,7 @@
 #
 ################################################################################
 
-SQUEEZELITE_VERSION = 71c012ad9ba102feb95823b7b9dc17e5305689c7
+SQUEEZELITE_VERSION = 3fbf7ea5288fad99e76800ee566771bbbce49299
 SQUEEZELITE_SITE = $(call github,ralph-irving,squeezelite,$(SQUEEZELITE_VERSION))
 SQUEEZELITE_LICENSE = GPL-3.0
 SQUEEZELITE_LICENSE_FILES = LICENSE.txt
