diff --git a/util-linux/chrt.c b/util-linux/chrt.c
index 45459d940..28e65457c 100644
--- a/util-linux/chrt.c
+++ b/util-linux/chrt.c
@@ -6,7 +6,7 @@
  * Licensed under GPLv2 or later, see file LICENSE in this source tree.
  */
 //config:config CHRT
-//config:	bool "chrt (4.4 kb)"
+//config:	bool "chrt (4.7 kb)"
 //config:	default y
 //config:	help
 //config:	Manipulate real-time attributes of a process.
